website/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "scripts": {
-    "prestart": "cp ../docs/cli.md.template ../docs/cli.md && node ../cli.js markdown-help >> ../docs/cli.md",
+    "prestart": "node lib/build.js",
```

Comment:
Doesn't this run `build.js` from the `website` directory, in which case `../docs/cli.md` path is incorrect?  Perhaps we should `chdir` within `build.js` for safety in any case (like we recently added to `screenshotter`).

Reply:
@edemaine `docs` is at the same level of `website`, so `../docs/cli.md` is correct path. I'm not sure cases of running `build.js` directly.